apps/site/src/app/(main)/apply/sections/Form/Form.tsx

```diff
@@ -93,11 +91,7 @@ export default function Form() {
 			encType="multipart/form-data"
 			onSubmit={handleSubmit}
 		>
-			<BasicInformation />
-			<SchoolInformation />
-			<ProfileInformation />
-			<ResumeInformation />
-			<AgeInformation />
+			{children}
```

Comment:
Issue: the main hacker application no longer has the form fields, so this component needs to be moved elsewhere, and a new component needs to be made for the hacker application. Would suggest using a different name (e.g. `apply/components/HackerApplicationForm.tsx`) so git can detect a rename of this file.